tests/wasmedge-build/run-tests.sh

```diff
@@ -13,7 +13,7 @@ make -j "$(nproc)"
 make install
 
 # Remove the installed crun to make sure the built crun is used
-rm -rf /usr/bin/crun
+rm -rf /usr/bin/crun /usr/local/bin/crun-wasm /usr/bin/crun-wasm
 ln -s /usr/local/bin/crun /usr/bin/crun
```

Comment:
Would be cleaner to just do `ln -sf /usr/bin/*```
Below?

Reply:
yeah this script is messy.  I've pushed a new version that doesn't create the symlinks and relies on `make install` to do it